tmt/base.py

```diff
@@ -220,6 +221,7 @@ def export(self, format_='yaml', keys=None, environment=None):
             data = dict()
             data['test'] = self.test
             data['path'] = self.path
+            data['require'] = " ".join(self.require)
```

Comment:
I think it should be a list :).

It gets processsed by `Prepare` which expects a list of packages to install...

Reply:
This is part of the export for `run.sh`. I don't think `run.sh` need to know about `require`. Or does it, @pvalena?

Reply:
I have no Idea on this one, as @lukaszachy 's added this.